src/contribute/feature-branches.md

```diff
@@ -0,0 +1,85 @@
+---
+layout: section
+title: "Beam Feature Branches"
+permalink: /contribute/feature-branches/
+section_menu: section-menu/contribute.html
+---
+
+# Feature Branches
+
+Some larger features are developed on a feature branch before being merged into
+`master`. In particular, this is often used for initial development of new
+components like SDKs or runners.
+
+It is important to note that the work on the branch is expected to be
```

Comment:
We expect the work on a feature branch to be _incomplete_, but it must not be lower quality. Code reviews for feature branches must have the same standards as code reviews for `master`. Once a feature branch is ready for merge to `master`, the set of changes will be too large to review in its entirety. Because of this, the code reviews during development must be thorough and trustworthy.